src/pages/dtrader/index.js

```diff
@@ -21,33 +30,29 @@ const items = [
 ]
 const trading = [
     {
-        title: (
-            <Localize
-                translate_text="Make smarter<0/>trading decisions"
-                components={[<br key={0} />]}
-            />
-        ),
-        subtitle: (
-            <Localize translate_text="Customise your chart with technical indicators and widgets - everything you need to make smarter trading decisions." />
+        title: localize('Make smarter trading decisions'),
```

Comment:
please use `<Localize />` instead of `localize` for indirect text render (e.g from array or object)